### PR TITLE
Reorganizes content renderer's description/info area + Other cleanup

### PR DESCRIPTION
**Pull request by christianmemije**
## Summary

* :lipstick: Reorganizes content renderer's description/info area
* Displays license description within a popover
* Navbar is now closed by default. Approved by @khangmach 
* Removes video player's spasm

### Before

before

### After

after

## Video players spasm before

smh

### CODE REVIEW

File: kolibri/plugins/learn/assets/src/views/content-page/index.vue [hunk 1]
```diff
@@ -50,25 +50,36 @@
       </icon-button>
     </assessment-wrapper>
 
-    <p class="page-description">{{ content.description }}</p>
+    <p>{{ content.description }}</p>
```
Comment (DXCanas) [inline kolibri/plugins/learn/assets/src/views/content-page/index.vue:53]: Nice cleanup

File: kolibri/plugins/learn/assets/src/views/content-page/index.vue [hunk 2]
```diff
@@ -50,25 +50,36 @@
...
 
-    <download-button v-if="canDownload" :files="content.files" class="download-button-left-align"/>
 
     <div class="metadata">
       <p v-if="content.author">
-        <strong>{{ $tr('author') }}: </strong>{{ content.author }}
+        {{ $tr('author') }}: {{ content.author }}
```
Comment (DXCanas) [inline kolibri/plugins/learn/assets/src/views/content-page/index.vue:58]: Are we purposely getting rid of the bold?

File: kolibri/plugins/learn/assets/src/views/content-page/index.vue [hunk 3]
```diff
@@ -50,25 +50,36 @@
...
       </p>
-      <p v-if="content.license">
-        <strong>{{ $tr('license') }}: </strong>{{ content.license }}
```
Comment (DXCanas) [inline kolibri/plugins/learn/assets/src/views/content-page/index.vue:62]: Nice

File: kolibri/plugins/learn/assets/src/views/content-page/index.vue [hunk 4]
```diff
@@ -50,25 +50,36 @@
...
+
+          <ui-popover trigger="licensetooltip" class="license-description">
+            {{ content.license_description }}
+          </ui-popover>
+        </template>
+
       </p>
+
       <p v-if="content.license_owner">
-        <strong>{{ $tr('copyrightHolder') }}: </strong>{{ content.license_owner }}
```
Comment (DXCanas) [inline kolibri/plugins/learn/assets/src/views/content-page/index.vue:68]: Again, are we supposed to be removing the bold?

File: kolibri/plugins/learn/assets/src/views/content-page/index.vue [hunk 5]
```diff
@@ -50,25 +50,36 @@
...
+        {{ $tr('copyrightHolder') }}: {{ content.license_owner }}
       </p>
     </div>
 
+    <download-button v-if="canDownload" :files="content.files" class="download-button"/>
```
Comment (DXCanas) [inline kolibri/plugins/learn/assets/src/views/content-page/index.vue:81]: hm. Looks like `download-button` was being required before, just not being used?

File: kolibri/plugins/learn/assets/src/views/content-page/index.vue (hunk 2 above)
Comment (christianmemije) [reply kolibri/plugins/learn/assets/src/views/content-page/index.vue:58]: yes.

File: kolibri/plugins/learn/assets/src/views/content-page/index.vue (hunk 4 above)
Comment (christianmemije) [reply kolibri/plugins/learn/assets/src/views/content-page/index.vue:68]: Yes, see https://projects.invisionapp.com/share/ZTA3FH2S2#/screens/232712226

File: kolibri/plugins/learn/assets/src/views/content-page/index.vue (hunk 5 above)
Comment (christianmemije) [reply kolibri/plugins/learn/assets/src/views/content-page/index.vue:81]: >  just not being used

Not sure what you mean with that.

The button was moved down based on the updated Learn mockups. 
https://projects.invisionapp.com/share/ZTA3FH2S2#/screens/232712226

**Comment by DXCanas** [comment]
Update for merge

**Comment by DXCanas** [comment]
Build failing due to python lint package. I think @ralphiee22 was working on this bug?

**Comment by ralphiee22** [comment]
I am working on this @DXCanas, but haven't figured out why the test is failing

**Pull request merged**